### DISCUSSION TITLE
Ceased Channels: PAC-12 Channels / June 30 2024

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#1225] [Show and tell]
Just 2 left Channels Oregon, Washington and Insider. All the rest ceased operations June 30 & 28 early on Xfinity. It's final moments https://youtu.be/gh3A7TmTuos.

**Comment by BellezaEmporium** [comment]
Can confirm. The first batch of channels ceased in June 30th.

Last live broadcast in May : https://www.youtube.com/watch?v=BJ4OelITkXo

With this, the Longhorn Network also shut down : https://www.youtube.com/watch?v=dI3ORvLKS-M (over the air only, now only airs online, and only for Texas games)

**Comment by R4IN2007** [comment]
> Can confirm. The first batch of channels ceased in June 30th.
> 
> Last live broadcast in May : https://www.youtube.com/watch?v=BJ4OelITkXo
> 
> With this, the Longhorn Network also shut down : https://www.youtube.com/watch?v=dI3ORvLKS-M (over the air only, now only airs online, and only for Texas games)

Longhorn Network launched as a streaming service.
